testing/ncids-css-testing/stories/components/nci-autocomplete/nci-autocomplete.scss

```diff
@@ -0,0 +1,98 @@
+@import '@nciocpl/ncids-css/scss/lib/normalize';
+@import '@nciocpl/ncids-css/scss/packages/required';
+@import '@nciocpl/ncids-css/scss/packages/global';
+@import '@nciocpl/ncids-css/scss/components/nci-header/nci-extended';
+@import 'uswds/dist/scss/base/accessibility';
+
+.nci-header-search {
+	justify-content: center;
+	margin: 0;
+	white-space: nowrap;
+	position: relative;
+  //modified here so it doesn't stretch all the way out
+	width: 420px;
+}
+
+.nci-header-search__search-button {
+  width: 77px;
+	border: 0;
+	border-bottom-left-radius: 0;
+	border-top-left-radius: 0;
+	font-size: 0.95rem;
+	margin: 0;
+	padding: 0.525rem 0.75rem;
+}
+
+//input
+.nci-header-search [type='search'] {
+  flex-grow: 1;
+  display: block;
+	border-width: 1px;
+	border-color: #757575;
+	border-style: solid;
+	border-right: none;
+	border-radius: 0;
+	box-sizing: border-box;
+	color: #1b1b1b;
```

Comment:
should color and border be color tokens?

Reply:
updated to reflect the updated _search and _nci-automplete partials.